.github/workflows/integ-test.yml

```diff
@@ -113,6 +123,7 @@ jobs:
         # python: [3.11]
         # test_name: [user_info]
         test_name: ${{ fromJson(needs.integ-matrix.outputs.matrix) }}
+        example_name: ${{ fromJson(needs.integration-prepare-env.outputs.matrix) }}
     steps:
```

Comment:
The matrix does generate matrix product. Now could would try to run a commands for each "test_name, example_name" combination. You need to copy both job definitions, and give them unique names (integ_bla -> example_bla).

Reply:
I created a separate job for running examples from the computed matrix - ``examples-run``.